.travis.yml

```diff
@@ -31,7 +31,7 @@ install:
 
   # install and build go-libp2p-daemon
   - curl -O -L -s -S https://raw.githubusercontent.com/status-im/nimbus-build-system/master/scripts/build_p2pd.sh
-  - bash build_p2pd.sh p2pdCache HEAD
+  - bash build_p2pd.sh p2pdCache v0.0.1
```

Comment:
Don't you want "v0.2.1" in here? That's the go-libp2p-daemon version we use in nim-beacon-chain.

Reply:
I'm playing it safe, but perhaps you're right, we can bump it to a higher version.